acceptance/acceptance_test.go

```diff
@@ -152,6 +152,11 @@ func testAccept(t *testing.T, InprocessMode bool, singleTest string) int {
 	uvCache := getUVDefaultCacheDir(t)
```

Comment:
We could cache this in actions as well. WDYT?

Reply:
It's hard to say if it's going to help (e.g. on Windows actually disable all cache helps because unpacking the whole cache is slow). It's also hard to measure, given high variance in different runs.

It might be beneficial it but I would not go for it right away.